test/Microsoft.VisualStudio.Web.CodeGeneration.EntityFrameworkCore.Test/TestModels/TestModel.cs

```diff
@@ -13,7 +15,8 @@ public static IModel CategoryProductModel
         {
             get
             {
-                var builder = new ModelBuilder(new CoreConventionSetBuilder().CreateConventionSet());
+                var sqlServerTypeMapper = new SqlServerTypeMapper(new RelationalTypeMapperDependencies()); 
```

Comment:
I assume these couldn't be easily mocked and that's why we needed to add that new package?